src/backendbot/api_routes.py

```
@@ -678,3 +539,399 @@ def get_logs(limit: int = 100) -> list[dict[str, Any]]:
+        # Backup de archivos de configuración comunes
+        config_files = [
+            "C:\\Windows\\System32\\drivers\\etc\\hosts",
+            os.path.expanduser("~\\AppData\\Local\\Microsoft\\Windows\\Shell\\LayoutModification.xml")
+        ]
+        
+        for file in config_files:
+            if os.path.exists(file):
+                shutil.copy2(file, backup_path)
+        
+        log_event(f"Backup creado en {backup_path}", notify_user=True)
+        return {"status": "ok", "message": f"Backup creado en {backup_path}"}
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)